src/Uno.UI/UI/Xaml/Controls/FlipView/FlipView.managed.cs

```diff
@@ -474,6 +474,11 @@ void InitializeScrollViewer()
 						m_tpScrollViewer.VerticalScrollMode = ScrollMode.Disabled;
 						m_tpScrollViewer.HorizontalScrollMode = ScrollMode.Enabled;
 					}
+#if HAS_UNO && __WASM__
+					// UNO workaround: the scrollability depends on both the ScrollMode and the ScrollBarVisibility, and by default the latter is Disabled.
+					m_tpScrollViewer.HorizontalScrollBarVisibility = m_tpScrollViewer.HorizontalScrollMode == ScrollMode.Enabled ? ScrollBarVisibility.Hidden : ScrollBarVisibility.Disabled;
+					m_tpScrollViewer.VerticalScrollBarVisibility = m_tpScrollViewer.VerticalScrollMode == ScrollMode.Enabled ? ScrollBarVisibility.Hidden : ScrollBarVisibility.Disabled;
```

Comment:
Is this `FlipView` specific? Regarding the comment I would imagine that we should update either the default `xxxScrollBarVisibility`, either the computation of the "scrollability"? Or is it just that we prefer to reduce the scope of that change to avoid regression?

Reply:
To limit the impact mostly.